packages/frontend/src/ee/features/aiCopilot/components/ChatElements/ScrollToBottom.tsx

```diff
@@ -67,11 +67,7 @@ const ThreadScrollToBottom = ({
             checkCurrentScrollPosition: true,
             behavior: 'auto',
         });
-    }, [
-        streamingState?.content,
-        streamingState.toolCalls.length,
-        scrollToBottom,
-    ]);
+    }, [streamingState?.content, streamingState?.toolCalls, scrollToBottom]);
```

Comment:
can't remember if the `.length` as part of the deps array was actually intentional 🤔